### ISSUE TITLE
Externalize some standards prefix mappings

### ISSUE THREAD

**Issue by MFSY** [BlueBrain/nexus-prov#14]
Let start with the following ones for now. These mappings need to be centralised in a module in nexus-kg in near future.

```json
{
  "skos": "http://www.w3.org/2004/02/skos/core#",
  "label": {
    "@id": "rdfs:label",
    "@type": "xsd:string"
  },
  "comment": {
    "@id": "rdfs:comment",
    "@type": "xsd:string"
  },
  "editorialNote": {
    "@id": "skos:editorialNote",
    "@type": "xsd:string"
  },
  "schema": "http://schema.org/",
  "prov": "http://www.w3.org/ns/prov#"
}
```

**Comment by MFSY** [comment]
Fixed in #32

**Comment by MFSY** [comment]
Fixed in #32
